### PR TITLE
add(delete fc command)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description
closes [#2749](https://github.com/Clinical-Genomics/cg/issues/2749)
### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b add-cli-delete-fc -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/delete/illumina_sequencing_run.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+from cg.constants.cli_options import DRY_RUN
+from cg.services.illumina.post_processing.housekeeper_storage import (
+    delete_sequencing_data_from_housekeeper,
+)
+
+
+@click.command("illumina-run")
+@DRY_RUN
+@click.option(
```
Comment (islean) [inline cg/cli/delete/illumina_sequencing_run.py:11]: Would it make more sense to have this as a `click.argument`? Difference would be `cg delete illumina-run <flow_cell_id>` vs `cg delete illumina-run --flow-cell-id <flow_cell_id>`

File: cg/cli/delete/illumina_sequencing_run.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    type=str,
+    multiple=False,
+    help="Give the flow cell id of the run to delete from statusdb and housekeeper.",
+)
+@click.pass_context
+def delete_illumina_run(context: click.Context, dry_run: bool, flow_cell_id: str):
+    """Delete an Illumina sequencing run"""
+    if dry_run:
+        click.echo("Dry run: Would delete the Illumina sequencing run")
+        return
```
Comment (islean) [inline cg/cli/delete/illumina_sequencing_run.py:23]: This dry run does not seem to add any value

**Review by islean — APPROVED** [review]
Looks good to me! 🔥

File: cg/cli/delete/illumina_sequencing_run.py (hunk 1 above)
Comment (karlnyr) [reply cg/cli/delete/illumina_sequencing_run.py:11]: I think production would say yes to this 👍 Thoughts @Clinical-Genomics/prodbioinfo ?+

Comment (beatrizsavinhas) [reply cg/cli/delete/illumina_sequencing_run.py:11]: Agree!

Comment (Karl-Svard) [reply cg/cli/delete/illumina_sequencing_run.py:11]: ^

Comment (ChrOertlin) [reply cg/cli/delete/illumina_sequencing_run.py]: Fixed

File: cg/cli/delete/illumina_sequencing_run.py (hunk 2 above)
Comment (ChrOertlin) [reply cg/cli/delete/illumina_sequencing_run.py]: True I removed it

**Comment by ChrOertlin** [comment]
tested

**Pull request merged**